src/Raven.Server/Program.cs

```diff
@@ -32,10 +32,13 @@ public class Program
     {
         private static readonly Logger Logger = LoggingSource.Instance.GetLogger<Program>("Server");
 
-        public static int Main(string[] args)
+        public static unsafe int Main(string[] args)
         {
             NativeMemory.GetCurrentUnmanagedThreadId = () => (ulong)Pal.rvn_get_current_thread_id();
 
+            Lucene.Net.Util.UnmanagedStringArray.Segment.AllocateMemory = NativeMemory.AllocateMemory;
```

Comment:
Also set this is RachisConsensusTestBase and TestBase

Reply:
done